examples/provider-components/image_versions_patch.yaml

```diff
@@ -20,19 +8,7 @@ spec:
     spec:
       containers:
         - name: manager
-          image: gcr.io/k8s-staging-cluster-api/cluster-api-controller:v0.3.16
----
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: capi-kubeadm-bootstrap-controller-manager
-  namespace: capi-webhook-system
-spec:
-  template:
-    spec:
-      containers:
-        - name: manager
-          image: gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller:v0.3.16
+          image: gcr.io/k8s-staging-cluster-api/cluster-api-controller:master
```

Comment:
and `k8s.gcr.io/cluster-api/cluster-api-controller:v0.4.0` here

Reply:
See [my comment above](https://github.com/metal3-io/cluster-api-provider-metal3/pull/254#discussion_r683264242)